config/initializers/instance_name.rb

```diff
@@ -3,4 +3,4 @@
 
 # Human-friendly name for this signon instance. This is used when generating
 # reminder emails that need to disambiguate between instances.
-Rails.application.config.instance_name = nil
+Rails.application.config.instance_name = "development"
```

Comment:
This file isn't overwritten in production, so this would be used in production. Maybe add something like `Rails.env.development? ? "development" : nil`